README.md

```diff
@@ -81,11 +81,18 @@ the service, you can use the following command, which will start the service on
 mage run
 ```
 
-Or, you can run docker-compose yourself:
+Or, you can run docker-compose yourself, building from source:
 ```shell
 cd build && docker-compose up --build
 ```
 
+To use the pre-published images:
+```shell
+cd build && docker-compose up -d
```

Comment:
I think we might change this instruction to "cd build && docker-compose -f docker-compose-published.yml up -d" which will run the new yml file